be/src/column/json_column.cpp

```diff
@@ -0,0 +1,57 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "column/json_column.h"
+
+#include "util/hash_util.hpp"
+#include "util/mysql_row_buffer.h"
+
+namespace starrocks::vectorized {
+
+int JsonColumn::compare_at(size_t left, size_t right, const starrocks::vectorized::Column& rhs,
```

Comment:
left and right are so common, suggest to use {left, right}_index

Reply:
fixed